src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css

```diff
@@ -213,7 +213,7 @@
 	list-style: none;
 	margin: 0;
 	line-height: 24px;
-	padding-left: 0;
+	padding-left: 1px;
```

Comment:
Out of curiosity, what is this one solving for? Looking good otherwise 👍

Reply:
There are left outline issues otherwise:
![PixelSnap 2023-07-24 at 22 58 20@2x](https://github.com/microsoft/vscode/assets/2276355/bafb3103-b19d-4eb1-ac57-2d21e5b8614d)

Reply:
Actually, there is a minor issue with the corners here. I will see if I can fix it...

Reply:
Ok, I'll await that change. Ideally the child elements (buttons) aren't fixing their own positioning issues with padding/margin and the container does that instead.

Reply:
I think the best option seems to be removing the background of the incomplete part of the progress bar, it already has a very similar color to the button background. The `outline-offset: -1px;` rule prevents small bleeding issues around the edges and because the progress bar color is the same as the outline this works but not if the color is different as with the incomplete progress bar color.

Reply:
Will this work across the different themes?